filters/src/test/java/io/camunda/zeebe/process/test/filters/logger/RecordStreamLoggerTest.java

```diff
@@ -15,16 +15,56 @@
  */
 package io.camunda.zeebe.process.test.filters.logger;
 
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.camunda.zeebe.process.test.api.RecordStreamSource;
 import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.RecordType;
+import io.camunda.zeebe.protocol.record.RecordValue;
 import io.camunda.zeebe.protocol.record.ValueType;
+import io.camunda.zeebe.protocol.record.intent.Intent;
+import io.camunda.zeebe.protocol.record.value.BpmnElementType;
+import io.camunda.zeebe.protocol.record.value.ImmutableDeploymentRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableErrorRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableIncidentRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableJobBatchRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableJobRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableMessageRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableMessageStartEventSubscriptionRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableMessageSubscriptionRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableProcessEventRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableProcessInstanceCreationRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableProcessInstanceRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableProcessInstanceResultRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableProcessMessageSubscriptionRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableTimerRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableVariableDocumentRecordValue;
+import io.camunda.zeebe.protocol.record.value.ImmutableVariableRecordValue;
+import io.camunda.zeebe.protocol.record.value.deployment.ImmutableDeploymentResource;
+import io.camunda.zeebe.protocol.record.value.deployment.ImmutableProcessMetadataValue;
 import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
 import java.util.Map;
 import java.util.function.Function;
 import org.assertj.core.api.SoftAssertions;
 import org.junit.jupiter.api.Test;
 
 class RecordStreamLoggerTest {
 
+  private static final Map<String, Object> TYPED_TEST_VARIABLES = new HashMap<>();
```

Comment:
:x: I am not so sure these tests add much value. I think it would be better to make `logVariables(...)` in `RecordStreamLogger` protected and test this method in isolation. 

if you still want to keep those kinds of tests I would propose to go deep into jqwik and make "Does not throw an exception" a property and then refactor this to spend most of the time generating aribtrary records. I think this will be a better investment going forward